uwblueprint/social-diversity-for-children
# Created Criminal Check Admin Page

### Notion ticket link

[Criminal Record Check approval](https://www.notion.so/uwblueprintexecs/Criminal-Record-Check-approval-2fbfd762ac8e400c9ff1f4aa5ebdf28c)



### Implementation description

- Implemented according to the design on Figma
- We don't have a date modified field so that line has been removed from the file
- Added null state for approval since default should be pending, not declined

### Steps to test

1. Go to the admin/criminal-check page and play around, most of the buttons do nothing except for the approval button on the file card, which should change the approval state on click.

### What should reviewers focus on?

- Check to see that the singular mutation works properly
- File downloads correctly

### Checklist


![image](https://user-images.githubusercontent.com/53587708/140559959-75933099-4d3d-44a8-aec0-691b23e37c57.png)
